CMakeLists.txt

```diff
@@ -0,0 +1,180 @@
+SET(PACKAGE osm2pgsql)
+SET(PACKAGE_NAME osm2pgsql)
+
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8.7)
+
+PROJECT(osm2pgsql)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+
+SET(DATA_DIR \".\")
+
+OPTION(BUILD_TESTS "Build with tests" OFF)
+
+if("${PROJECT_SOURCE_DIR}" STREQUAL "${PROJECT_BINARY_DIR}")
+  message(FATAL_ERROR "In-source builds are not allowed, please use a separate build directory like `mkdir biuld && cd build && cmake ..`")
+endif()
+
+execute_process (COMMAND git describe OUTPUT_VARIABLE PACKAGE_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+message(STATUS "Building osm2pgsql ${PACKAGE_VERSION}")
+
+if (NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE=Release)
+endif()
+
+# Just in case user installed RPMs from http://yum.postgresql.org/
+list(APPEND PostgreSQL_ADDITIONAL_SEARCH_PATHS /usr/pgsql-9.3 /usr/pgsql-9.4)
```

Comment:
Not sure. By default even Ubuntu can not find installed PostgreSQL. Is it better to include all pathes in cmakelists.txt to avoid "not compile" tickets?